lib/msal-common/src/unifiedCache/UnifiedCacheManager.ts

```diff
@@ -0,0 +1,123 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { InMemoryCache, JsonCache } from "./utils/CacheTypes";
+import { Separators } from "../utils/Constants";
+import { AccessTokenEntity } from "./entities/AccessTokenEntity";
+import { IdTokenEntity } from "./entities/IdTokenEntity";
+import { RefreshTokenEntity } from "./entities/RefreshTokenEntity";
+import { AccountEntity } from "./entities/AccountEntity";
+import { ICacheStorage } from "../cache/ICacheStorage";
+import { Deserializer } from "./serialize/Deserializer";
+import { Serializer } from "./serialize/Serializer";
+import { AccountCache } from "../utils/MsalTypes";
+
+export class UnifiedCacheManager {
+    // Storage interface
+    private inMemoryCache: InMemoryCache;
+    private cacheStorage: ICacheStorage;
+
+    constructor(cacheImpl: ICacheStorage) {
+        this.cacheStorage = cacheImpl;
+        this.inMemoryCache = this.generateInMemoryCache(
+            this.cacheStorage.getSerializedCache()
+        );
+    }
+
+    /**
+     * set the cache memory
```

Comment:
nit: Consider only leaving comments where they add extra information. For example this comment is exactly the same thing as the setter name - it adds not extra value and just clutters the code.

Reply:
This is mostly function description, to follow the norms describing each function. Choosing to comment a few functions and not others looks odd to me. We can clean up unneeded comments within code but I would still like each function have a little desc.